must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -114,3 +113,23 @@ echo "collecting dump of oc get obc all namespaces" | tee -a  "${BASE_COLLECTION
 echo "collecting dump of oc get volumereplication all namespaces" | tee -a  "${BASE_COLLECTION_PATH}/gather-debug.log"
 { oc get volumereplication --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/vr_all_namespaces"
 { oc adm --dest-dir="${BASE_COLLECTION_PATH}/namespaces/all/" inspect volumereplication --all-namespaces --"${SINCE_TIME}"; } >> "${BASE_COLLECTION_PATH}/gather-debug.log" 2>&1
+
+# For StorageSystem of all namespaces
+echo "collecting dump of oc get storagesystem all namespaces" | tee -a  "${BASE_COLLECTION_PATH}/gather-debug.log"
+{ oc get storagesystem --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/ss_all_namespaces"
+{ oc describe storagesystem --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/ss_all_namespaces"
+{ oc get storageclusters --all-namespaces -o yaml; } >> "${BASE_COLLECTION_PATH}/namespaces/all/ss_all_namespaces.yaml"
```

Comment:
Aren't we already collecting this?
It should probably be a separate block starting with `echo "collecting dump of storageclusters across all namespaces"`

Reply:
```suggestion
{ oc get storageclusters --all-namespaces -o yaml; } >> "${BASE_COLLECTION_PATH}/namespaces/all/storagesystem_yaml_all_ns"
```